powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
By the way, what’s the point in this script at all? `powerline-lemonbar-multimonitor` seems to be capable of handling single-monitor configuration fine.